python/ray/experimental/serve/examples/echo_split.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
This file/example and the previous one with rollback could probably be combined and it would make more sense.